docs/source/release/v6.2.0/muon.rst

```diff
@@ -11,54 +11,53 @@ Frequency Domain Analysis
 New Features
 ############
 
-- The Frequency Domain Analysis interface now allows you to perform a sequential fit using the Sequential Fitting tab.
-- The Sequential Fitting tab allows you to choose the type of dataset you want to fit.
-- New `Maxent Dual Plot` option has been added to the plotting. This will show the reconstructed data and the
-  raw data together, along with the frequency domain data.
+- The Frequency Domain Analysis interface now allows you to perform a sequential fit using the :ref:`Sequential Fitting tab <muon_sequential_fitting_tab-ref>`.
+- The :ref:`Sequential Fitting tab <muon_sequential_fitting_tab-ref>` allows you to choose the type of dataset you want to fit.
+- **New** :ref:`Maxent Dual Plot <Maxent_Dual_Plot-ref>` **option has been added to the plotting. This will show the reconstructed data and the
+  raw data together, along with the frequency domain data.**
 
-Bug Fixes
-#########
+.. image::  ../../images/maxent_dual_plot.PNG
```

Comment:
I don't think this image is being found. Try using lowercase `.png`

Reply:
Can you see the other images in the notes? They both have .PNG rather than lowercase.

Reply:
Yes I can see the other two. Its because the other two have the extension `.PNG` but the new image has it lowercase as `.png`